.github/workflows/coding-standards.yml

```diff
@@ -0,0 +1,26 @@
+name: "coding standards"
+
+on: ["pull_request", "push"]
+
+jobs:
+  unit-tests:
+    name: "coding standards"
+
+    runs-on: "ubuntu-latest"
+
+    steps:
+      - name: "checkout"
+        uses: "actions/checkout@v2"
+
+      - name: "installing PHP"
+        uses: "shivammathur/setup-php@v2"
+        with:
+          php-version: "7.4"
+          ini-values: memory_limit=-1
+          tools: composer:v2, cs2pr
+
+      - name: "installing php-cs-fixer"
```

Comment:
Can't you use the PHP CS Fixer provided by setup-php?

Reply:
ah! forgot about `tools`, will do 👍

Reply:
done